src/index.ts

```diff
@@ -149,6 +155,61 @@ export const addKeypairToEnvFile = async (
   await appendFile(fileName, `\n${variableName}=${secretKeyString}`);
 };
 
+export interface initializeKeypairOptions {
+  envFileName?: string; // .env file path
+  variableName?: string; // Variable name to call the secret key in the environment file
+  airdropAmount?: number; // Optional amount to airdrop in lamports
+  minimumBalance?: number; // Optional minimum balance to maintain in lamports
+  keypairPath?: string; // Optional path to a keypair file, if not provided, will look for the keypair in the environment
+}
+
+/**
+ * Loads a keypair from the environment or generates a new one if not found and writes it to the environment,
+ * then requests an airdrop if the balance is below the minimum balance
+ *
+ * @param connection Connection to the Solana cluster
+ * @param options initializeKeypairOptions
+ * - envFileName: .env file path
+ * - variableName: Variable name to call the secret key in the environment file
+ * - airdropAmount: Optional amount to airdrop in lamports
+ * - minimumBalance: Optional minimum balance to maintain in lamports
+ * - keypairPath: Optional path to a keypair file, if not provided, will look for the keypair in the environment
+ * @returns A keypair with funds loaded
+ */
+export const initializeKeypair = async (
+  connection: Connection,
+  options?: initializeKeypairOptions,
+): Promise<Keypair> => {
+  let {
+    envFileName,
+    variableName,
+    airdropAmount,
+    minimumBalance,
+    keypairPath,
+  } = options || {};
+
+  let signer: Keypair;
```

Comment:
Just use the name `keypair`. Our function is called `initializeKeypair` the thing it returns should be `keypair`.

Reply:
Deal